onmt/opts.py

```diff
@@ -603,6 +622,8 @@ def translate_opts(parser):
     group.add('--output', '-output', default='pred.txt',
               help="Path to output the predictions (each line will "
                    "be the decoded sequence")
+    group.add('--report_align', '-report_align', action='store_true',
```

Comment:
How will this work? Will it be dumped in the translation file (`-output`) or do we maybe need to add another file `-output_align`?

Reply:
We just paste the alignment output behind translation, separated by ` ||| `, ex: 'This is a translation ||| 1-1 2-2 3-3'